boot/cmdline_test.go

```diff
@@ -209,5 +209,5 @@ func (s *kernelCommandLineSuite) TestComposeCommandLineManagedHappy(c *C) {
 	c.Assert(cmdline, Equals, "panic=-1 snapd_recovery_mode=recover snapd_recovery_system=20200314")
 	cmdline, err = boot.ComposeCommandLine(model)
 	c.Assert(err, IsNil)
-	c.Assert(cmdline, Equals, "panic=-1")
+	c.Assert(cmdline, Equals, "panic=-1 snapd_recovery_mode=run")
```

Comment:
shouldn't this be the reverse order, i.e. `panic=-1` is <static> from snapd and `snapd_recovery_mode=run` is "mode" ? From this morning I have the order as: "mode" "static" "extras" a la

```
snapd_recovery_mode=run console=ttyS0 console=tty1 panic=-1 foo=bar_from_gadget
snapd_recovery_system=20200624 snapd_recovery_mode=install console=ttyS0 console=tty1 panic=-1
snapd_recovery_system=20200624 snapd_recovery_mode=install console=ttyS0 console=tty1 panic=-1
```

although arguably `console=ttyS0 console=tty1` could be the "extras" from the gadget to appease folks who don't want their UC systems to slow down waiting for serial output and `panic=-1` should be static but that's not material here

Reply:
Also a specific device could have special-purpose serial ports and the serial console could be elsewhere.